FoxHound.Client/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TrainingApp",
   "main": "index.js",
   "scripts": {
-    "dev": "parcel src/index.html"
+    "dev": "parcel src/index.html",
+    "start": "parcel src/index.html"
```

Comment:
It looks like you added a 'start' command (which is fine), but consider having the start command just call the dev command (or visa versa). Then we don't have to maintain 2 of the same script if it changes. You should be able to do something like "start": "npm run dev"